mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
@choldgraf let's make `delays` private as `_delays`, since users probably shouldn't care about the sample delays